mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,50 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """
+    All the info that gets validated
+    has been set by the projects
+    bat files.
+    If you are surtain that the info
+    is incorrect, talk to
+    project Supervisor with haste !
+
+    """
+
+    label = "Validate Project Edit Info"
+    optional = True
+    order = pyblish.api.ValidatorOrder
+    hosts = ["maya"]
+
+    def process(self, context):
+        from maya import cmds
+
+        sceneIn = cmds.playbackOptions(query=True, animationStartTime=True)
+        sceneOut = cmds.playbackOptions(query=True, animationEndTime=True)
+        sceneFPS = {
+            "12fps" : 12,
+            "game"  : 15,
+            "film"  : 24,
+            "pal"   : 25,
+            "ntsc"  : 30,
+            "show"  : 48,
+            "palf"  : 50,
+            "ntscf" : 60}.get(cmds.currentUnit(query=True, time=True))
```

Comment:
Do not use `.get` unless you are prepared to handle `None`.

In this case, `None` may be returned which would produce a difficult-to-understand message for the user below.

> `The FPS is set to 'None'`